linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_mac_modification/ansible/shared.yml

```diff
@@ -0,0 +1,46 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+
+#
+# Inserts/replaces the rule in /etc/audit/rules.d
+#
+- name: Search /etc/audit/rules.d for other MAC modification audit rules
+  find:
+    paths: "/etc/audit/rules.d"
+    recurse: no
+    contains: "-k MAC-policy$"
+    patterns: "*.rules"
+  register: find_mac
```

Comment:
Note many customers do not use the ``-k`` field. Suggest not basing any remediation on its existence.

Reply:
(or they rename to something more appropriate for their environment)